setup.py

```diff
@@ -63,11 +63,10 @@ def read_version_py(infname):
     #   Twisted[conch] also depends on cryptography and Twisted[tls]
     #   transitively depends on cryptography.  So it's anyone's guess what
     #   version of cryptography will *really* be installed.
+    "cryptography >= 2.6",
```

Comment:
I don't know the reason for the prior `< 40` but that leads me to wonder if this should be `> 41` or something...? Or does the pyOpenSSL part cover what we need?

Reply:
pyOpenSSL will make sure the appropriate `cryptography` version it needs is installed, yeah. (There's some weird "not these minor versions" due to these particular constants.)